db/migrate/20210804123238_create_screenings.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateScreenings < ActiveRecord::Migration[6.1]
+  def change
+    create_table :screenings do |t|
+      t.datetime :date
```

Comment:
Consider that client may not want to have screenings without a date in his cinema ;) Consider the same in CinemaHalls migration